Objects/complexobject.c

```diff
@@ -14,6 +14,8 @@
 #include "pycore_pymath.h"        // _Py_ADJUST_ERANGE2()
 
 
+#define _PyComplexObject_CAST(op)   ((PyComplexObject *)(op))
```

Comment:
I wanted to keep a small diff but I can do that (as you can see, I've opened 20 PRs doing the same mechanical changes...)

Reply:
If you do this mechanically (i.e. with sed), I would guess you should take some extra steps to prevent other changes in same file.

Perhaps, it's better to not introduce a new macro and stick with `(PyComplexObject *)` casts.  Or change everything.

Reply:
When Victor and I started fixing #111178, we introduced a macro so that we can (in the future), possibly add type-checks easily. By "mechanical" changes, I manually changed everything (but everytime I repeated the same steps)